boards/px4/fmu-v3/init/rc.board

```diff
@@ -44,6 +44,7 @@ fi
 
 # External I2C bus
 hmc5883 -C -T -X start
+qmc5883 -C -X start
```

Comment:
The driver fails to start on this command and has to be manually started in the console.